index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
- [ ] [OPTIONAL] 🚀  To help boost your commits 🤓 , kindly consider using **imperative mood in the subject line**. Example – ```Add fix for dark mode toggle state``` instead of ```Added or fixed```. Here is an awesome [article](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) you can sneak peek at once you're free.